### DISCUSSION TITLE
Hardware on device like VM

### DISCUSSION THREAD

**Discussion by juliouzzzz** [bb-Ricardo/netbox-sync#189] [Q&A]
Hi,

Is there a way to sync hardware (RAM, DISK, CPU) from host to device ?
I can't find fields "Ressources" on devices like we could find on VM :
![image](https://user-images.githubusercontent.com/86098048/168295826-83be3dc4-f336-4d71-bcbe-5081097fdf25.png)

Does check_redfish do it ?
May i have to create custom field on device ?

Thanks

**Comment by bb-Ricardo** [comment]
Yes, this somehow does not exists for devices. And yes, you can use the check_redfish source to add this data.

It will be added as Inventory, not as custom attributes.

maybe that would also be an idea for the redfish source to add summary of cpu, ram and disk as custom fields to devices 🤔

**Reply by juliouzzzz** [reply]
Ok thanks, i've never try inventory part but ok i'll try with API to put data into inventory.
But yes "summary ressources" could be great as custom field. should i open an FR ? ;)

Julien

**Reply by bb-Ricardo** [reply]
sure. Yes please :D
